### ISSUE TITLE
Possible bug in VES

### ISSUE THREAD

**Issue by GiovanniBussi** [plumed/plumed2#306]
@valsson I noticed there are scaring messages on the travis log in some of the ves tests. The test is not failing, but perhaps there is some other problem.

I think the problem came out in between these two builds:
https://travis-ci.org/plumed/plumed2/jobs/286450888
https://travis-ci.org/plumed/plumed2/jobs/286897299
(search the log for 'rt-md_le-1d-mpi-4-replicas-diff-frictions'). Can you check if this is related to the pull request we recently merged?

Thanks!

Giovanni

PS In any case, we should understand why the whole suite is not reporting this as a failure...

**Comment by valsson** [comment]
@GiovanniBussi Thanks for letting me know! I had not noticed this. 

I think I have found the problem, I made a stupid mistake in opening a dummy file output, used /dev/null/ instead of /dev/null, such that the file is not opened correctly. Then the code probably segfaults when it tries to close the file at the end. This would explain why it doesn't fail, the problem is at the end after all the output files have been written, therefore the file check in the regtests is fine. Also, the problem was hidden on my mac laptop, only after looking carefully into the plumed output did I notice it. 

I have submitted a fix onto travis. I will make a PR onto the v2.4 branch if that fixes the problem.
